event-pages/event-1.html

```diff
@@ -0,0 +1,240 @@
+			rel="stylesheet"
+		/>
+
+		<!-- Vendor CSS Files -->
+		<link href="../assets/vendor/aos/aos.css" rel="stylesheet" />
+		<link
+			href="../assets/vendor/bootstrap/css/bootstrap.min.css"
+			rel="stylesheet"
+		/>
+		<link
+			href="../assets/vendor/bootstrap-icons/bootstrap-icons.css"
+			rel="stylesheet"
+		/>
+		<link
+			href="../assets/vendor/glightbox/css/glightbox.min.css"
+			rel="stylesheet"
+		/>
+		<link
+			href="../assets/vendor/swiper/swiper-bundle.min.css"
+			rel="stylesheet"
+		/>
+
+		<!-- Template Main CSS File -->
+		<link href="../assets/css/style.css" rel="stylesheet" />
+		<link href="../assets/css/arabic.css" rel="stylesheet" />
+
+		<!-- =======================================================
+  * Template Name: Imperial - v5.7.0
+  * Template URL: https://bootstrapmade.com/imperial-free-onepage-bootstrap-theme/
+  * Author: BootstrapMade.com
+  * License: https://bootstrapmade.com/license/
+  ======================================================== -->
+	</head>
+
+	<body>
+		<!-- ======= Header ======= -->
+		<header id="header" class="d-flex align-items-center">
+			<div class="container d-flex align-items-center justify-content-between">
+				<a href="index.html" class="logo mr-auto"
+					><img src="../assets/img/landing/logo1.png" alt=""
+				/></a>
+				<!-- Uncomment below if you prefer to use a text logo -->
+				<!-- <h1 class="logo mr-auto"><a href="index.html">OSCSA</a></h1> -->
+
+				<nav id="navbar" class="navbar">
+					<ul>
+						<li>
+							<a class="nav-link scrollto active" href="../index.html">Home</a>
+						</li>
+						<li><a class="nav-link scrollto" href="#about">About</a></li>
+						<li>
+							<a class="nav-link scrollto" href="#services">Open Science</a>
+						</li>
+						<li>
+							<a class="nav-link scrollto" href="#testimonials"
+								>UNESCO Recommendation</a
+							>
+						</li>
+						<li>
+							<a class="nav-link scrollto" href="#materilas">Materials</a>
+						</li>
+						<li><a class="nav-link scrollto" href="#contact">Contact</a></li>
+						<li>
+							<a href="ar-index.html">عربي <i class="bi bi-globe"></i></a>
+						</li>
+					</ul>
+					<i class="bi bi-list mobile-nav-toggle"></i>
+				</nav>
+				<!-- .navbar -->
+			</div>
+		</header>
+		<!-- End Header -->
+
+		<main id="main">
+			<!-- ======= Event Details Section ======= -->
+			<section id="event-details">
+				<div class="container" data-aos="fade-up">
+					<div class="col-md-12">
+						<h3 class="section-title">
+							Leveraging Open Science in Machine Learning and Bioinformatics
+						</h3>
+						<div class="section-title-divider"></div>
+					</div>
+					<div class="event-body">
+						<img
+							src="../assets/img/event/event1.png"
+							alt="event-image"
+							class="event-image"
+						/>
+						<div class="mt-5">
+							<p class="event-description">
```

Comment:
Add a "Description"  text before the describtion content`<h4> Description <h4/>`

Reply:
understood